TinCan.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 2013
+VisualStudioVersion = 12.0.31101.0
+MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
Can we revert these changes, at least the change from VS 2012 to VS2013. Though I don't really know what they are for. Any explanation or just the IDE doing automatic things? In general we should avoid making these kinds of changes if possible.

Reply:
Neither VS2017 nor Rider require the lines VisualStudioVersion...  and MinimumVisualStudioVersion...
In some projects it makes sense to keep such constraints in the sln files, such as those which depend on the UML-to-Code transformations which were included in previous VS versions but was removed in VS 2017.

I suggest the additions to be discarded while the deletion to be kept.